mycroft/skills/skill_updater.py

```diff
@@ -45,228 +101,85 @@ class SkillUpdater:
                                 with the mycroft core system and handle
                                 commands.
     """
-    _installed_skills_file_path = None
-    _msm = None
 
     def __init__(self, bus=None):
+        self.selene_skills = SeleneDeviceSkills()
+        self.post_manifest(True)
+
+        self.installed_skills = set()
+
+        # below are unused, only for api backwards compat
         self.msm_lock = ComboLock(get_temp_path('mycroft-msm.lck'))
         self.install_retries = 0
         self.config = Configuration.get()
         update_interval = self.config['skills'].get('update_interval', 1.0)
         self.update_interval = int(update_interval) * ONE_HOUR
-        self.dot_msm_path = os.path.join(self.msm.skills_dir, '.msm')
-        self.next_download = self._determine_next_download_time()
-        self._log_next_download_time()
-        self.installed_skills = set()
+        self.dot_msm_path = "/tmp/.msm"
```

Comment:
Is this an internal variable?

Reply:
since it wasnt private i can not remove it, just to keep the api compatible. but a bunch of those are just placeholder to avoid syntax errors, need to keep the api compatible even if nothing in ovos-core itself uses those variables

they are mock values that should be safe if something is consuming them

Reply:
I missed that this was an existing param. Maybe we should start annotating params left around for compat?

Reply:
i left the code comment to indicate all those are unused and just for api compatibility, but we could probably get this in somewhere more visible.... even for community documentation

Reply:
my bad, you did annotate that